cmd/mtaBuild.go

```diff
@@ -563,3 +582,22 @@ func getAbsPath(path string) string {
 	}
 	return filepath.FromSlash(abspath)
 }
+
+func getPurl(mtaYaml string) string {
```

Comment:
This part we have repeated in https://github.com/SAP/jenkins-library/blob/master/cmd/mavenBuild.go#L277 and https://github.com/SAP/jenkins-library/blob/master/pkg/npm/publish.go#L229

Can we move the logic to https://github.com/SAP/jenkins-library/blob/master/pkg/piperutils/cyclonedxBom.go

And use the same in all places?

Reply:
[4cf86a0](https://github.com/SAP/jenkins-library/pull/5166/commits/4cf86a06b75b0d1243eb6629897bc8ca4cd016fc)